packages/webpack/test/scaffold.js

```diff
@@ -65,7 +74,13 @@ function convertToMap(input, path = '') {
   return result
 }
 
-function runScript(code, globals = {}) {
+const defaultGlobals = () => ({
+  console: {
+    log: () => {},
+  },
+})
+
+function runScript(code, globals = defaultGlobals()) {
```

Comment:
remind me if this calls `defaultGlobals()` every time `runScript` is called without a second parameter, or if it's called just once